modules/xfeatures2d/include/opencv2/xfeatures2d.hpp

```diff
@@ -179,6 +179,55 @@ class CV_EXPORTS_W LATCH : public Feature2D
     CV_WRAP static Ptr<LATCH> create(int bytes = 32, bool rotationInvariance = true, int half_ssd_size = 3, double sigma = 2.0);
 };
 
+/*
+* BEBLID Descriptor
+*/
+
```

Comment:
Please remove this comment block (it not not needed).

Other similar places will be removed later (its usage is not consistent).

Reply:
done